spec/i18n_spec.rb

```diff
@@ -15,24 +15,7 @@
 ].sort.freeze
 
 # These are keys with mismatch interpolation for specific locales
-ALLOWED_INTERPOLATION_MISMATCH_LOCALE_KEYS = [
-  # need to be fixed
-  'zh.account_reset.pending.confirm',
-  'zh.account_reset.pending.wait_html',
-  'zh.account_reset.recovery_options.check_webauthn_platform_info',
-  'zh.doc_auth.info.exit.with_sp',
-  'zh.idv.cancel.headings.exit.with_sp',
-  'zh.idv.failure.exit.with_sp',
-  'zh.telephony.account_reset_notice',
-  'zh.telephony.confirmation_otp.voice',
-  'zh.two_factor_authentication.account_reset.pending',
-  'zh.user_mailer.account_reset_granted.intro_html',
-  'zh.user_mailer.account_reset_request.header',
-  'zh.user_mailer.account_reset_request.intro_html',
-  'zh.user_mailer.in_person_verified.next_sign_in.without_sp',
-  'zh.user_mailer.in_person_verified.subject',
-  'zh.user_mailer.new_device_sign_in.info',
-].sort.freeze
+ALLOWED_INTERPOLATION_MISMATCH_LOCALE_KEYS = [].freeze
```

Comment:
Should we just remove this? I don't think we'll ever intentionally want a mismatch?